clients/corral/corral.go

```diff
@@ -95,6 +94,42 @@ func CreateCorral(ts *session.Session, corralName, packageName string, debug boo
 	return msg, nil
 }
 
+// CreateCorral creates a corral taking the corral name, the package path, and a debug set so if someone wants to view the
+// corral create log
+func CreateCorral(ts *session.Session, corralName, packageName string, debug, cleanup bool) (*exec.Cmd, error) {
+	ts.RegisterCleanupFunc(func() error {
+		return DeleteCorral(corralName)
+	})
+
+	args := []string{"create"}
+	if !cleanup {
+		args = append(args, skipCleanupFlag)
+	}
+	if debug {
+		args = append(args, debugFlag)
+	}
+	args = append(args, corralName, packageName)
+	logrus.Infof("Creating corral with the following parameters: %v", args)
+
+	corralCommand := exec.Command("corral", args...)
+
+	var b bytes.Buffer
+	corralCommand.Stdout = &b
+	corralCommand.Stderr = &b
+	b.Bytes()
```

Comment:
This is unused. Can we remove it?

Reply:
removed

Reply:
This still seems like the case, I can't find where is this `b` used, should it log it out info level?

Reply:
not this part though Caleb `b.Bytes()` this isn't doing anything. It's returning a bytes array you and your not setting it to any new variable https://pkg.go.dev/bytes#Buffer.Bytes

Reply:
oh I see. Okay I can remove this line sorry I thought you meant this whole block